int8 address # The starting address to write to
int8 last    # The last address to write to
bool value   # The value to write to the specified address